mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelProvider.kt

```diff
@@ -65,7 +65,7 @@ object MvRxViewModelProvider {
                     "with a create method returning a non-null ViewModel."
             } else {
                 "${MvRxViewModelFactory::class.java.simpleName} must have primary constructor with a " +
```

Comment:
I think the main point of the issue was that `${MvRxViewModelFactory::class.java.simpleName} ` is used, which is not the right class,

`viewModelClass` . should be used instead